vignettes/howtogenefilter.Rmd

```diff
@@ -0,0 +1,140 @@
+---
+title: "Using the genefilter function to filter genes from a microarray dataset"
+author:
+- name: "Khadijah Amusat"
+  affiliation: "Vignette translation from Sweave to R Markdown / HTML"
+date: "`r format(Sys.time(), '%B %d , %Y')`"
+output:
+    BiocStyle::html_document:
+    number_sections: true
+    toc: true
+    toc_depth: 4
+package: genefilter
+vignette: >
+  %\VignetteIndexEntry{Using the genefilter function to filter genes from a microarray}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+# Introduction {#introduction .unnumbered}
+
+The `r Biocpkg("genefilter")` package can be used to filter (select) genes from a microarray dataset according to a variety of different filtering mechanisms. Here, we will consider the example dataset in the `sample.ExpressionSet` example from the `r Biocpkg("Biobase")` package. This experiment has 26 samples, and there are 500 genes and 3 covariates. The covariates are named `sex`, `type` and `score`. The first two have two levels and the last one is continuous.
+
+```{r, closeg, message = FALSE}
+library(Biobase) 
+library(genefilter)
+data(sample.ExpressionSet)
+varLabels(sample.ExpressionSet)
+table(sample.ExpressionSet$sex)
+table(sample.ExpressionSet$type)
+data(sample.ExpressionSet)
```

Comment:
Remove line 30, which looks like a duplicate of line 26. It's not in the [PDF](https://bioconductor.org/packages/release/bioc/vignettes/genefilter/inst/doc/howtogenefilter.pdf).